sound/soc/sof/ipc4-topology.c

```diff
@@ -2138,6 +2138,14 @@ static int sof_ipc4_route_setup(struct snd_sof_dev *sdev, struct snd_sof_route *
 		return 0;
 	}
 
+	if (!src_fw_module || !sink_fw_module) {
+		/* The NULL module will print as "(efault)" */
+		dev_err(sdev->dev, "source %s or sink %s widget didn't set up properly\n",
```

Comment:
nit-pick: the use of "didn't" is a bit weird, maybe "widget not set-up properly"

Reply:
@plbossart I replaced it with "weren't," is that better?